eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -98,9 +98,14 @@ stages:
                 deploy:
                   steps:
                     - checkout: self
+                    - template: /eng/pipelines/templates/steps/stage-artifacts.yml
```

Comment:
I think this has been removed now with @mitchdenny's recent work.

Reply:
You're absolutely correct 👍

Reply:
Updated.